apps/valkyrie/lib/valkyrie/broadway.ex

```diff
@@ -89,12 +90,22 @@ defmodule Valkyrie.Broadway do
           error: :failed_schema_validation,
           reason: inspect(reason)
         )
+        
+        decoded_message_data = Jason.decode!(message_data.value)
+        ingestion_id = decoded_message_data["ingestion_id"]
```

Comment:
How certain are we that this actually exists? I don't think messages have ingestion_id in valkyrie. Did we hotpatch to confirm?

Reply:
We did, yes
The ingestion id exists in the message_data.value portion of the message inside a JSON string

Reply:
Awesome. Lets add it to the DeadLetter.process then - Didn't realize we had access to that variable here.